cli/samples/multi-cluster.yaml

```diff
@@ -0,0 +1,31 @@
+# sample-config.yaml
+terraformState: local  # local, cloud
```

Comment:
Did you intend to push this file?

Reply:
I did. I wanted to have a folder of examples for folks to use. Should we put it somewhere else?

Reply:
no this is fine! just making sure you didn't push it by accident